src/examples/PickerExamplePage.tsx

```diff
@@ -37,22 +37,22 @@ export const PickerExamplePage: React.FunctionComponent<{}> = () => {
         },
       ]}>
       <Example title="A simple Picker." code={example1jsx}>
-        <Picker style={{height: 50, width: 100}}>
+        <Picker style={{height: 32, width: 200}}>
```

Comment:
This change is fine because it makes the numbers more correct. But later we should reevaluate whether these controls can go without a specified height or not and get the right default.

Reply:
If my memory serves me I believe any time I specified control dimensions in gallery it was because the control wouldn't render unless height/width were specified.

Reply:
It's certainly been an issue before (see [RNW#5437](https://github.com/microsoft/react-native-windows/issues/5437). But I think @rectified95 addressed some of those and the others we hit should be tracked. It's worth checking in with what controls need and which don't.

Reply:
As far as I remember, the community Picker didn't need dimensions. I may be wrong, though